src/clj/athens/self_hosted/components/fluree.clj

```diff
@@ -21,10 +21,18 @@
           component)
         (do
           (log/info "Starting Fluree connection, servers" servers)
-          (let [conn (fdb/connect servers)]
+          (let [;; Wrap conn in an atom so we can reconnect without restarting component.
+                conn-atom    (atom nil)
+                reconnect-fn (fn []
+                               (when-some [conn @conn-atom]
+                                 (fdb/close conn))
+                               (reset! conn-atom (fdb/connect servers)))
+                comp         {:conn-atom    conn-atom
```

Comment:
`:conn-atom` & `:reconnect-fn` should be part `defrecord`.
for now it's still having `conn` in `defrecord`